src/protocols/dhcpv4/encode.c

```diff
@@ -36,11 +36,7 @@
 
 static inline bool is_encodable(fr_dict_attr_t const *root, VALUE_PAIR *vp)
 {
-	if (!vp) return false;
-	if (vp->da->flags.internal) return false;
-	if (!fr_dict_parent_common(root, vp->da, true)) return false;
-
-	return true;
+	return vp && !vp->da->flags.internal && fr_dict_parent_common(root, vp->da, true);
```

Comment:
I'd prefer to use the previous code.  Experience shows it's easier to read.

Reply:
I'm convinced. Back to the previous code.